src/main/java/terminator/command/DeadlineCommand.java

```diff
@@ -0,0 +1,51 @@
+package terminator.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import terminator.task.DeadlineTask;
+import terminator.task.Task;
+
+/**
+ * Concrete class representing a command to create a DeadlineTask.
+ */
+public class DeadlineCommand extends Command {
+    private static final String ERR_MSG = """
+            Deadline description cannot be empty.\n
+            Usage: deadline <description> /by dd/MM/yyyy HHmm""";
+
+    public DeadlineCommand(String input) {
+        super(input);
+    }
+
+    /**
+     * Creates a new DeadlineTask with the given input and adds it to the task list.
+     *
+     * @param todoList The task list.
+     * @throws TerminatorException if the description of the deadline task is empty.
+     */
+    @Override
```

Comment:
When writing Javadocs for overridden methods, the `@inheritDoc` tag can be used to reuse the header comment from the parent method but with further modifications e.g., when the method has a slightly different behavior from the parent method. _Else, if f I am not wrong:_ The JavaDoc tool will use the comment from the overriding method.